Source/WebCore/css/parser/CSSPropertyParserConsumer+Timeline.cpp

```diff
@@ -150,5 +152,41 @@ RefPtr<CSSValue> consumeViewTimelineInset(CSSParserTokenRange& range, const CSSP
     });
 }
 
+static bool isAnimationRangeKeyword(CSSValueID id)
+{
+    return identMatches<CSSValueNormal, CSSValueCover, CSSValueContain, CSSValueEntry, CSSValueExit, CSSValueEntryCrossing, CSSValueExitCrossing>(id);
+}
+
+RefPtr<CSSValue> consumeAnimationRange(CSSParserTokenRange& range, const CSSParserContext& context, TimelineRangeStartOrEnd isStart)
```

Comment:
```suggestion
RefPtr<CSSValue> consumeAnimationRange(CSSParserTokenRange& range, const CSSParserContext& context, TimelineRangeStartOrEnd startOrEnd)
```

nit: please be consistent with enum name, whatever it ends being

Reply:
Fixed.